api/data_test.go

```diff
@@ -39,8 +39,86 @@ func TestGetCacheTimeEndpoint(t *testing.T) {
 				}
 			},
 		}
+
+		dataStoreAPI := setupAPIWithStore(ctx, dataStoreMock)
+
+		Convey("When an existing cache time is requested with its ID", func() {
+			request := httptest.NewRequest(http.MethodGet, baseURL+testCacheID, http.NoBody)
+			responseRecorder := httptest.NewRecorder()
+			dataStoreAPI.Router.ServeHTTP(responseRecorder, request)
+
+			Convey("The status code should be 200", func() {
+				So(responseRecorder.Code, ShouldEqual, http.StatusOK)
+			})
+		})
+	})
+}
+
+func TestUpdateExistingCacheTimeRetursn401WhenAuthIsOnAndAuthTokenIsMissiing(t *testing.T) {
+	Convey("Given an existing cache time", t, func() {
```

Comment:
Whether a CacheTime resource exists or not is irrelevant for the purposes of this test: because the user is unauthorised, we won't be even calling MongoDB. So this test should be refactored and all the unneeded parts removed.